specification/e-referrals-service-api.yaml

```diff
@@ -496,7 +496,44 @@ paths:
           $ref: 'components/schemas/responses/Forbidden.yaml'
         '500':
           $ref: 'components/schemas/responses/InternalServerError.yaml'
+  /R4/v1/PractitionerRole:
+    get:
+      security:
+        - bearerAuth: []
+      description: |
+        ## Overview
+        
+        Use this endpoint to retrieve a list of available e-RS roles (i.e. business functions) at each organisation where the user works.
```

Comment:
As written, it sounds like you might be able to use this EP to find out the role info for an arbritrary user.
It might be better to say "the authenticated user", or "the user on whose behalf the API is called" to clarify that this isn't the case.

Reply:
Text part of the overview comes from collaboration between us and NHS. I think its still subject to final review but for now we should just use the approved text as is.